pkg/test/cluster/environment.go

```diff
@@ -23,13 +23,20 @@ import (
 
 // Environment a cluster-based environment for testing.
 type Environment struct {
-	T testing.TB
+	T                    testing.TB
+	IstioSystemNamespace string
+	AppNamespace         string
 }
 
 // Deploy pushes the given helm configuration to the cluster.
 func (e *Environment) Deploy(c *helm.Chart) {
 }
 
+// DeployApps pushes the pilot test apps to the cluster.
```

Comment:
Nit: Remove pilot from comment. I think the notion of Test/Mock Apps should be generic and usable by other components if needed. (Implementation-wise, they are already that way I believe, so this is mostly about marketing).

Reply:
done.